service/src/main/java/org/ehrbase/service/ValidationServiceImp.java

```diff
@@ -131,6 +135,44 @@ public void check(Composition composition) throws Exception {
         check(composition.getArchetypeDetails().getTemplateId().getValue(), composition);
     }
 
+    @Override
+    public void check(EhrStatus ehrStatus) {
+
+        //case of a system generated ehr
+        if (ehrStatus == null)
+            return;
+
+        //first, check the built EhrStatus using the general Archie RM-Validator
+        List<RMObjectValidationMessage> rmObjectValidationMessages = RM_OBJECT_VALIDATOR.validate(ehrStatus);
+
+        if (!rmObjectValidationMessages.isEmpty()){
+            StringBuilder stringBuilder = new StringBuilder();
+            for (RMObjectValidationMessage rmObjectValidationMessage: rmObjectValidationMessages){
+                stringBuilder.append(rmObjectValidationMessage.toString());
+                stringBuilder.append("\n");
```

Comment:
I observed a wrong input body (f.e. without `name`) returns the message from this string builder like intended, but the new lines are just literal `\n`, instead of a line break.